src/styles/style.css

```diff
@@ -0,0 +1,92 @@
+@font-face {
+  font-family: Avenir;
+  src: url(../fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(../fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+body {
+  margin: 0;
+}
+
+.search-bars {
+  width: 100%;
+}
+
+.search-bar {
+  display: flex;
+  margin-inline: 8px;
+  align-items: center;
+}
+
+.search-bar__input {
+  width: 100%;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px #3d4e611a;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+
+  font-family: Avenir, sans-serif;
```

Comment:
you can use inherit property here, also define font-family for all document by using html selector